src/server/game/Entities/Item/Item.h

```diff
@@ -152,11 +152,11 @@ enum EnchantmentSlot
     PRISMATIC_ENCHANTMENT_SLOT      = 6,                    // added at apply special permanent enchantment
     MAX_INSPECTED_ENCHANTMENT_SLOT  = 7,
 
-    PROP_ENCHANTMENT_SLOT_0         = 7,                    // used with RandomSuffix
```

Comment:
Seems strange for me to allow only some enchantment slots for RandomSuffix or RandomProperty knowing that they can't coexist together (an item can't have RandomSuffix AND RandomProperty).